p2panda-rs/src/storage_provider/traits/log_store.rs

```diff
@@ -56,8 +56,14 @@ pub trait LogStore<StorageLog: AsStorageLog> {
 
 #[cfg(test)]
 pub mod tests {
+    // TEST MODULE NOTES //
+    //
+    // In this module we are implementing the `LogStore` trait on `SimplestStorageProvider` just
```

Comment:
Im not sure if I understand this comment: aren't all tests written inside the `tests` module solely for testing purposes? 🤔

Reply:
Yeh, it clear without the comment why the traits are being implemented here in the test module? It's for us to test the methods with default implementations, and also so we can test the higher level methods on `StorageProvider`? This is what I was trying (obviously not very well) to explain.

Reply:
Traits in a library crate are weird indeed. You provide an interface to the outside you don't use yourself (inside the library), except of maybe tests and partly in our some default implementations. I think its safe to assume that this is "normal" when testing traits in a library.